### PR TITLE
fix(tab): tab should render as a div instead of a button element

### PR DESCRIPTION
**Pull request by eljefe223**
Closes: #888 
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-components-react-base/src/tabs/tabs.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ class Tabs extends Foundation<ITabsHandledProps & ITabsManagedClasses, ITabsUnha
                     "aria-selected": this.state.activeId === tabItem.props.id,
                     onClick: this.handleClick,
                     onKeyDown: this.handleKeyDown,
-                    tabIndex: this.state.activeId !== tabItem.props.id ? -1 : 0
+                    tabIndex: this.state.activeId !== tabItem.props.id ? -1 : 1
```
Comment (chrisdholt) [inline packages/fast-components-react-base/src/tabs/tabs.tsx]: Should be 0 instead of 1.

**Comment by chrisdholt** [comment]
Can you update your merge message?

File: packages/fast-components-react-base/src/tabs/tabs.tsx (hunk 1 above)
Comment (janechu) [reply packages/fast-components-react-base/src/tabs/tabs.tsx]: @eljefe223 to add to this it's helpful to look at https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex which is a more accurate source for this kind of information.

**Comment by janechu** [comment]
please change the PR message to conform to our conventional commits

**Comment by eljefe223** [comment]
Whoops sorry I missed the merge message. Will update.

**Pull request merged**